pkgs/applications/networking/instant-messengers/pidgin-plugins/tdlib-purple/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, tdlib, pidgin, libwebp, libtgvoip } :
+
+stdenv.mkDerivation rec {
+  pname = "tdlib-purple";
+  version = "0.7.6";
+
+  src = fetchFromGitHub {
+    owner = "ars3niy";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1inamfzbrz0sy4y431jgwjfg6lz14a7c71khrg02481raxchhzzf";
+  };
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
```

Comment:
I'm pretty sure this is already included by nix.

Reply:
Yes, this is set by the setup hook.

Reply:
good to know :)